src/jahia2wp.py

```diff
@@ -24,6 +24,8 @@
   jahia2wp.py veritas               <csv_file>                      [--debug | --quiet]
   jahia2wp.py inventory             <wp_env> <path>                 [--debug | --quiet]
   jahia2wp.py extract-plugin-config <wp_env> <wp_url> <output_file> [--debug | --quiet]
+  jahia2wp.py list-plugins          <wp_env> <wp_url>               [--debug | --quiet]
```

Comment:
est ce que tu pourrais rajouter un exemple d'appel dans le README ?

par exemple dans la section 2. 🎉 configuring the website with supported plugins and the EPFL theme...

Reply:
J'ai ajouté une section 3 en fait.